active/001-tabular-msa.md

```diff
@@ -0,0 +1,82 @@
+---
+Feature name: tabular-msa
+Start date: 2015-08-14
+RFC PR: <leave empty>
+scikit-bio issue: <leave empty>
+---
+
+# Summary
+
+Replace the current experimental `skbio.alignment.Alignment` class with a
+`TabularMSA` class, providing an API for manipulating tabular (row-column)
```

Comment:
Hadn't occurred to us. Anyone else have thoughts on this? I think `Matrix` might imply too much about the implementation (who knows, we might just store diffs from a consensus to save memory internally).

I also think putting the `MSA` at the end makes it read a little better, so `MatrixMSA` would be my vote should we go with `Matrix` as the descriptor.